inbox/api/jsc_api.py

```diff
@@ -179,3 +181,23 @@ def create_account():
         except NotSupportedError as e:
             resp = simplejson.dumps({ 'message': str(e), 'type': 'custom_api_error' })
             return make_response((resp, 400, { 'Content-Type': 'application/json' }))
+
+@app.route('/provider_from_email', methods=['get'])
+def provider_from_email():
+    g.parser.add_argument('email', required=True, type=bounded_str, location='args')
+    args = strict_parse_args(g.parser, request.args)
+
+    try:
+        provider_name = provider_from_address(args['email'])
+        provider_info = providers[provider_name] if provider_name != 'unknow' else 'unknow'
```

Comment:
There's a small typo here that will affect the logic: `unknow` (missing 'n') 

https://github.com/jobscore/sync-engine/blame/master/inbox/util/url.py#L136